receiver/tlscheckreceiver/config.go

```diff
@@ -28,31 +29,49 @@ type Config struct {
 }
 
 type targetConfig struct {
-	URL string `mapstructure:"url"`
+	Host string `mapstructure:"host"`
```

Comment:
Please use one of the standard configs like https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confignet/confignet.go#L127

Reply:
I changed it to use `confignet.TCPAddrConfig`